src/devices/Mcp3428/Config.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mcp3428
+{
+    /// <summary>
+    /// Possible gain values of the ADC
+    /// </summary>
+    public enum GainEnum : byte { X1 = 0, X2 = 1, X4 = 2, X8 = 3 }
+
+    /// <summary>
+    /// Possible operation modes of the ADC
+    /// </summary>
+    public enum ModeEnum : byte { OneShot = 0, Continuous = 16 }
```

Comment:
Can we remove `Enum` suffix? we usually don't use such prefixes. Perhaps just `Gain` and `Mode` or `AdcGain`/`AdcMode`